samples/matter/lock/src/app_task.cpp

```diff
@@ -261,6 +271,13 @@ CHIP_ERROR AppTask::Init()
 	/* Initialize lock manager */
 	BoltLockMgr().Init(LockStateChanged);
 
+	/* Register Door Lock test event trigger */
+#ifdef CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_REGISTER_DEFAULTS
```

Comment:
I'd say in the case of adding a custom event trigger the `CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS` must be enough.